pkg/controller/argocd/rolebinding_test.go

```diff
@@ -86,17 +86,3 @@ func TestReconcileArgoCD_reconcileClusterRoleBinding(t *testing.T) {
 	clusterRoleBinding = &rbacv1.ClusterRoleBinding{}
 	assert.NilError(t, r.client.Get(context.TODO(), types.NamespacedName{Name: expectedName}, clusterRoleBinding))
 }
-
-func TestReconcileArgoCD_reconcileRoleBinding_application_controller(t *testing.T) {
```

Comment:
Please add back this test (make any adjustment necessary).

Reply:
This test was added to test if the hooks were executed when a new role binding was being created but now we have removed the hook itself so we don't need this test case at all. 
We don't call `applyReconcilerHook` while reconciling the `roleBinding` so i think it is not needed anymore.